VisualizationBase/src/ViewItemManager.h

```diff
@@ -44,19 +48,27 @@ class VISUALIZATIONBASE_API ViewItemManager
 		ViewItemManager(Scene* scene);
 		~ViewItemManager();
 
-		void addViewItem(ViewItem* view, QPoint position = QPoint(-1, -1));
 		ViewItem* newViewItem(const QString name = QString(), QPoint position = QPoint(-1, -1));
 		ViewItem* viewItem(const QString name);
+		void addViewItem(ViewItem* view, QPoint position = QPoint(-1, -1));
```

Comment:
This class has way too many public methods and its API could be improved:
* It seems like we have not one but three ways to create views:
 * `newViewItem`
 * `addViewItem`
 * `createIfNotExist`
 * We should have exactly one public method to do this
* `viewItemsAsList` seems unnecessary to me
* We should reconsider having public `saveView`/`loadView` methods.
 * `loadView` is only called by the manager itself. Why not make it private and make the API leaner?
* `saveView` is only called with the current view as an argument. I think it's better if we make the current implementation private and have a new public method `saveCurrentView` or perhaps even `saveAllViews`. It might even be a good idea to save these automatically every time something changes on the view.